backend/chain.go

```diff
@@ -368,6 +368,84 @@ func (g *gnoZenaoChain) UserAddress(userID string) string {
 	return gnolang.DerivePkgAddr(g.userRealmPkgPath(userID)).String()
 }
 
+// CreatePoll implements ZenaoChain
+func (g *gnoZenaoChain) CreatePoll(userID string, req *zenaov1.CreatePollRequest) error {
+	userRealmPkgPath := g.userRealmPkgPath(userID)
+	eventPkgPath := g.eventRealmPkgPath(req.EventId)
+	feedID := gnolang.DerivePkgAddr(eventPkgPath).String() + ":main"
+	options := ""
+	for _, option := range req.Options {
```

Comment:
should we use Join to avoid : ```"opt1", "opt2", ```

Reply:
> should we use Join to avoid : `"opt1", "opt2", ` ?

i think it's nit, and using join would give ``"opt1, opt2"`` instead of ``"\"opt1\", \"opt2\""`` i think